classes/pdf/PDF.php

```diff
@@ -115,6 +115,8 @@ public function __construct($objects, $template, $smarty, $orientation = 'P')
         if (count($this->objects) > 1) { // when bulk mode only
             $this->send_bulk_flag = true;
         }
+
+        $this->getFilename();
```

Comment:
Why do you call this method as you don't fetch the return value ?

Reply:
The method sets the class property. This change was requested by @matks

Reply:
In this case, the `getFilename` as a bad name as it define the variable `$filename`.

Reply:
@Progi1984 ok so what do you want? Shall I also create a setter method?